var/spack/repos/builtin/packages/arm/package.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import re
+
+import llnl.util.tty as tty
+import spack.compiler
+import spack.util.executable
+
+
+class Arm(Package):
+    """Arm Compiler combines the optimized tools and libraries from Arm
+    with a modern LLVM-based compiler framework.
+    """
+
+    homepage = "https://developer.arm.com/tools-and-software/embedded/arm-compiler"
```

Comment:
I think arm compiler's home page is https://developer.arm.com/tools-and-software/server-and-hpc/compile

Reply:
@tkameyama is correct. The link in the package currently is more geared toward mobile and embedded applications.

Reply:
Fixed in 76f3697 @LPettey-Arm Can you please double check that:
```console
$ spack external find arm
```
correctly identifies arm compilers installed on a system? Also if you could share how these compilers are laid out in a filesystem tree that would help me add detection tests in a later PR. Thanks!